Tasks/JavaToolInstallerV0/FileExtractor/JavaFilesExtractor.ts

```diff
@@ -2,6 +2,7 @@ import fs = require('fs');
 import path = require('path');
 import taskLib = require('vsts-task-lib/task');
 import tr = require('vsts-task-lib/toolrunner');
+import toolLib = require('vsts-task-tool-lib/tool');
```

Comment:
We've been changing this to `import * as toolLib from 'vsts-task-tool-lib/tool';` in our tasks.  That is a stricter way to import (for example, the `toolLib` module object will be immutable, whereas import-require basically gives you a plain JS object).  Can you change that for this file?